src/shared/useScreenReaderEnabled.ts

```diff
@@ -0,0 +1,21 @@
+import {useEffect, useState} from 'react';
+import {AccessibilityInfo, AccessibilityEvent} from 'react-native';
+
+export function useScreenReaderEnabled() {
+  const [screenReaderEnabled, setEnabled] = useState(false);
```

Comment:
I think this won't work with autofocus since the value of this prop is only used on mount and will always be false. We'd need to keep this state in some global state since we can't get the current value synchronously.

Reply:
ah ya, good point.